packages/graphql-hooks/src/useClientRequest.ts

```diff
@@ -246,6 +247,17 @@ function useClientRequest<
           })
         }
 
+        if (
+          revisedOpts.onSuccess &&
```

Comment:
undefined is not an instance of Function, should we remove this check?

Reply:
made in https://github.com/nearform/graphql-hooks/pull/928/commits/4ea3bd85457943db1b1032920ce4d5d03901d171